dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -99,9 +98,9 @@
           %td On function   
```

Comment:
Hmm, maybe something like "Show on window event"?  This field is the name of the event that will trigger the callout when fired on `window`.